### PR TITLE
ENH Replace ADMIN permissions with less permissions in Behat test

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Description
Replace user with "Admin" permissions to user with "Editor" permissions.

### Parent issue
- https://github.com/silverstripe/silverstripe-behat-extension/issues/220

### CODE REVIEW

File: tests/behat/features/blog.feature [hunk 1]
```diff
@@ -3,17 +3,17 @@ Feature: Create a blog
   I want to create a blog
 
   Background:
-    Given the "group" "EDITOR group" has permissions "CMS_ACCESS_LeftAndMain"
+    Given the "group" "EDITOR group" has permissions "Access to 'Pages' section"
```
Comment (ChloeSartorelli) [inline tests/behat/features/blog.feature]:
```suggestion
    Given the "group" "EDITOR" has permissions "Access to 'Pages' section"
```
The name of the group shouldn't have "group" in it

File: tests/behat/features/blog.feature [hunk 2]
```diff
@@ -3,17 +3,17 @@ Feature: Create a blog
...
 
     # Login then logout to created member
-    And I am logged in with "EDITOR" permissions
-    And I go to "/Security/login"
-    And I press the "Log in as someone else" button
-    And I am logged in with "ADMIN" permissions
+    And I am logged in as a member of "EDITOR" group
+    # And I go to "/Security/login"
+    # And I press the "Log in as someone else" button
+    # And I am logged in with "ADMIN" permissions
```
Comment (ChloeSartorelli) [inline tests/behat/features/blog.feature]: Don't have commented out code like this. If it's not needed, just remove it. If you do remove it, remove the comment above as well.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: tests/behat/features/blog.feature (hunk 2 above)
Comment (sabina-talipova) [reply tests/behat/features/blog.feature]: DONE

File: tests/behat/features/blog.feature (hunk 1 above)
Comment (sabina-talipova) [reply tests/behat/features/blog.feature]: DONE

File: tests/behat/features/blog.feature [hunk 3]
```diff
@@ -3,19 +3,14 @@ Feature: Create a blog
...
 
   Background:
-    Given the "group" "EDITOR group" has permissions "CMS_ACCESS_LeftAndMain"
+    Given the "group" "EDITOR" has permissions "CMS_ACCESS_CMSMain"
     And I add an extension "SilverStripe\Widgets\Extensions\WidgetPageExtension" to the "Page" class
     And I add an extension "SilverStripe\Comments\Extensions\CommentsExtension" to the "Page" class
        
     And an "image" "Uploads/file1.jpg"
 
     # Login then logout to created member
```
Comment (ChloeSartorelli) [inline tests/behat/features/blog.feature]:
```suggestion
```
This comment refers to the part you deleted.

File: tests/behat/features/blog.feature [hunk 4]
```diff
@@ -35,21 +30,25 @@ Feature: Create a blog
...
+    When I go to "/admin/pages"
+    And I follow "New Blog"
     And I click the "Settings" CMS tab
-    And I press the "Save" button
     And I click the "Users" CMS tab
     And I wait for 3 seconds
     And I select "EDITOR" from "Editors"
     And I press the "Publish" button
-
-    # Logout
```
Comment (ChloeSartorelli) [inline tests/behat/features/blog.feature]: Why are you removing this comment?

File: tests/behat/features/blog.feature [hunk 5]
```diff
@@ -68,6 +67,7 @@ Feature: Create a blog
     And I click the "Post Options" CMS tab
     And I add "My Category" to the "Categories" tag field
     And I add "My Tag" to the "Tags" tag field
+    And I wait for 2 seconds
```
Comment (ChloeSartorelli) [inline tests/behat/features/blog.feature]: Why have you added this?

Comment (sabina-talipova) [reply tests/behat/features/blog.feature]: Removed. "Click Publish button" breaks test execution and throws alert message in local environment without this delay.

File: tests/behat/features/blog.feature (hunk 4 above)
Comment (sabina-talipova) [reply tests/behat/features/blog.feature]: DONE

File: tests/behat/features/blog.feature (hunk 3 above)
Comment (sabina-talipova) [reply tests/behat/features/blog.feature]: DONE

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**